webapp/src/main/webapp/src/app/sender/archive/sender-archive.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from "@angular/core";
+import { BaseService } from "../../shared/base.service";
+import { Observable } from "rxjs";
+import { SenderSyncArchive } from "./sender-archive";
+import { SenderSyncArchiveCountAndItems } from "./sender-sync-archive-count-and-items";
+
+const RESOURCE_NAME = 'sender/archive';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class SenderArchiveService extends BaseService<SenderSyncArchive> {
+
+	getArchiveCountAndItems(): Observable<SenderSyncArchiveCountAndItems> {
+		return this.getCountAndItems(RESOURCE_NAME);
+	}
+
```

Comment:
Isn't this supposed to be named `searchByPeriod`?

Reply:
No.